packages/react/src/components/cookieConsent/useCookies.ts

```diff
@@ -0,0 +1,28 @@
+import { useMemo } from 'react';
+
+import { ConsentObject, COOKIE_NAME, getCookieDomainFromUrl, parseConsents } from './cookieConsentController';
+import { createCookieController } from './cookieController';
+
+type ReturnType = {
```

Comment:
I wonder if we would wish to export this ReturnType? Would the users of the `useCookie` hook benefit from getting the ReturnType? If we do export it, we probably need to give it a more specific name.

Reply:
The returned type should be exposed to the IDE without exporting it. But might be good to export it if devs pass the prop somewhere.

Reply:
Exported as UseCookiesHookReturnType